src/CSI.IBTA.Administrator/Controllers/LoginController.cs

```diff
@@ -0,0 +1,75 @@
+using CSI.IBTA.Administrator.Models;
+using Microsoft.AspNetCore.Mvc;
+using System.Diagnostics;
+using System.Net;
+using CSI.IBTA.Administrator.Interfaces;
+using CSI.IBTA.Administrator.Extensions;
+using CSI.IBTA.Administrator.Endpoints;
+
+namespace CSI.IBTA.Administrator.Controllers
+{
+    public class LoginController : Controller
+    {
+        private readonly IAuthClient _client;
+        private readonly IJwtTokenService _jwtTokenService;
+
+        public LoginController(IAuthClient client, IJwtTokenService jwtTokenService)
+        {
+            _client = client;
+            _jwtTokenService = jwtTokenService;
+        }
+
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Login(LoginViewModel model)
+        {
+            if (!ModelState.IsValid)
+            {
+                return View("Index", model);
+            }
+
+            var response = await _client.PostAsync(model.ToDto(), AuthApiEndpoints.Auth);
+            if (!response.IsSuccessStatusCode)
+            {
+                if (response.StatusCode == HttpStatusCode.InternalServerError)
```

Comment:
LoginController should not handle 500 errors. 
We may need a separate page for handling them. 
Create a task in JIRA "Handle errors" and remind to discuss it on standup. 

 https://learn.microsoft.com/en-us/aspnet/core/fundamentals/error-handling?view=aspnetcore-8.0